application/features/analytics.feature

```diff
@@ -61,4 +61,41 @@ Feature: User
         And that user has an unverified u2f mfa record
       And I create a new user
     When I get the count of active users with a verified u2f mfa
-    Then the count of active users with a verified u2f mfa should be 1
\ No newline at end of file
+    Then the count of active users with a verified u2f mfa should be 1
+
+  Scenario: Get a count of active users with a password
+    Given that no mfas or users exist
+      And I create a new user with a password
+      And I create a new user with a password
+      And I create a new user without a password
+      And I create a new user without a password
```

Comment:
It's probably not a big deal, but it might be worth having a different number of users with and without a password in case we ever update that logic and get something backwards (like our recent `'yes'` acting like `0` scenario), just to help ensure this test would catch that.